### DISCUSSION TITLE
Is there a way to generalize blocks in a schematic?

### DISCUSSION THREAD

**Discussion by cs-deathmatch** [maruohon/litematica#836] [Q&A]
I'm not sure if that's the best way to phrase it, but basically i'm wondering if there's a way i can tell the mod "hey, these blocks don't matter, as long as they meet X Y or Z criteria", so that they don't get marked red in the overlay while i'm building it

for example, if i built the walls of a farm out of stone bricks, i could say "the stone bricks don't matter, as long as they're a solid block". If i used signs to block water, i could say "the wood type of these signs don't matter", "the type of stair here doesnt matter as long as it's not flammable", etc

**Comment by maruohon** [comment]
There isn't anything for that atm. There is a system planned for "valid substitutes", but that would likely be user configurable, so I'm not sure if you could define such things in the schematic file already. But I can see how that could be useful, so maybe it should be able to work that way as well.
